tests/screenshots/cases/software/basic/text.slint

```diff
@@ -0,0 +1,15 @@
+// Copyright © SixtyFPS GmbH <info@slint-ui.com>
+// SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-commercial
+
+TestCase := Window {
+    width: 64px;
+    height: 64px;
+
+    Text {
+        text: "Hello World";
+        wrap: word-wrap;
+        color: green;
+        font-size: 15px;
+        width: 100%;
+    }
+}
```

Comment:
May I suggest a slightly more complicated example to test more stuff per images, so we increase the test coverage.

```
TestCase := Window {
    width: 64px;
    height: 64px;
    VerticalLayout { 
        Text {
            text: "Hello Everyone";
            wrap: word-wrap;
            color: green;
            font-size: 10px;
        }
        Text {
            text: "Hello";
            color: #abcd;
            opacity: 0.9;
            font-size: 10px;
            horizontal-alignment: center;
        }
        Text {
            text: "Hello everyone";
            font-size: 10px;
            width: 100%;
            horizontal-alignment: right;
        }
    }
}
```

Reply:
I was thinking about that, too, but I'd prefer to get the basics implemented here first. But if you say we should go right for the complex example, then I'll work on that instead.

Reply:
To avoid a higher PNG churn in the repo, I've rebased it and included this as `test.slint`, with one modification: For the last text box, "Hello Everyone" is too long - I've shorted it to just "Everyone" so that the right alignment is visible.